ra/ra.go

```diff
@@ -1774,7 +1761,7 @@ func (ra *RegistrationAuthorityImpl) PerformValidation(
 	// Clock for start of PerformValidation.
 	vStart := ra.clk.Now()
 
-	if req.Authz == nil || req.Authz.Id == "" || req.Authz.Identifier == "" || req.Authz.Status == "" || req.Authz.ExpiresNS == 0 {
+	if req.Authz == nil || req.Authz.Id == "" || req.Authz.Identifier == "" || req.Authz.Status == "" || core.IsAnyNilOrZero(req.Authz.Expires) {
```

Comment:
Use this core.IsAnyNilOrZero to replace this whole giant set of `||`, not just one item in it. It takes variadic arguments for a reason!

Same on lines 2443 and 2660 of this file, and various places in the SA and WFE